src/gcp_scanner/scanner.py

```diff
@@ -32,15 +32,14 @@
 from googleapiclient import discovery
 from httplib2 import Credentials
 from .models import SpiderContext
+from typing import Union
 
-
-def is_set(config, config_setting):
+def is_set(config: Union[None,dict], config_setting: str) -> Union[dict,bool]:
```

Comment:
Replace `Union[None, X]` with `Optional[X]`

Reply:
Hi @ZetaTwo, I have made the requested change for Union[None, x]. Let me know if it's fine.